src/test/resources/org/zalando/nakadi/validation/invalid-json-schema-examples.json

```diff
@@ -147,9 +147,7 @@
         }
       }
     },
-    "errors": ["Invalid schema found in [#/definitions/paradox]: extraneous key [not] is not permitted",
-      "Invalid schema found in [#/items]: extraneous key [$ref] is not permitted",
```

Comment:
😅 yes... but I maybe should open a PR to the json-schema meta-schema repository with this same fix. It's a bug in the original file from the specs repo as well.